src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: "Avenir";
```

Comment:
It is better to put the values of font-family, colors and sizes into separate variables and reuse them.